app/src/main/java/de/tum/in/tumcampusapp/api/Helper.java

```diff
@@ -102,6 +103,17 @@ public Response intercept(Chain chain) throws IOException {
         };
     }
 
+    /**
+     * encodes an url
+     *
+     * @param pUrl input url
+     * @return encoded url
+     */
+    public static String encodeUrl(String pUrl) {
+        String url = pUrl.replace("/", ""); //remove slashes in queries as this breaks the url
```

Comment:
Yea please do. Someone else might come along and think to use this function without understanding that this is some corner case in some class.